site/en/codelabs/openthread-simulation-posix/index.lab.md

```diff
@@ -836,7 +836,7 @@ Done
 
 ####  5. Validate connectivity
 
-Quit `ot-ctl` by using **Ctrl+D** and on your host machine's command line,
+Quit `ot-ctl` by using either **Ctrl+D** or `exit` command, and on your host machine's command line,
```

Comment:
Please make the same changes on the openthread-simulation codelab

https://github.com/openthread/ot-docs/blob/main/site/en/codelabs/openthread-simulation/index.lab.md?plain=1#L892

Reply:
We'll update the other Codelab in a separate PR so this one isn't blocked.